imperial_coldfront_plugin/gpfs_client.py

```diff
@@ -219,16 +258,33 @@ def _create_fileset_set_quota(
     fileset_name: str,
     owner_id: str,
     group_id: str,
-    path: str,
+    path: Path,
```

Comment:
It might be more elegant, and require less assumptions on the specific structure of the `path`, which might change in the future, to provide 2 arguments for it instead of only one:
- `department_path`: up to, and including, the department.
- `project_path`: the project-specific part of the path, including the PI. 

This way you can use one or the other, or a combination of both if needed `path = department_path / project_path`, as needed, and you are clearer of the overall structure of the paths.

Reply:
Likewise a good shout.